joomla/joomla-cms
# [5.2] Add deleted files and folders from debugbar dependency update to script.php

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) updates the lists of files and folders to be deleted on update in file `administrator/components/com_admin/script.php` in the 5.2-dev branch in preparation for the next 5.2.0-alpha3 release.

All files and folders added by this PR are deleted due to the update of the composer dependency "maximebf/debugbar" from version "v1.21.3" to version "v1.22.3", where the fontawesome fonts used by the debug bar have been moved from font files into base64 encoded data in the css file.

This and several other composer dependencies have been updated with the 5.2.0-alpha2 release of today with this commit for the version bump: https://github.com/joomla/joomla-cms/commit/e7f3f5bcee8a81bfc0967c1635cb5e86bdac0bfc

Normally such dependency updates have to be made with a public PR here in this repo so people can review and test, and I can make a PR like this one here now before the release is built.

This time it has been made different, so the updates from this PR here will go into the next alpha release. That's sufficient in this case as the deleted files are just some obsolete font files which are not used anymore, so nothing's broken.

But in general it would be better to do it advance like I said.

### Testing Instructions

Code review, or if you want to do a real test:

Update a 5.2.0-alpha1 to 5.2.0-alpha2 to get the actual result, and update la 5.2.0-alpha1 to the patched package created by Drone for this PR to get the expected result.

Here you can find the patched update package or custom update URL created by Drone for this PR: __will be added soon__

### Actual result BEFORE applying this Pull Request

The files and folders added by this PR to file `administrator/components/com_admin/script.php` are still present after the update.

### Expected result AFTER applying this Pull Request

The files and folders added by this PR to file `administrator/components/com_admin/script.php` have been removed with the update.

### Link to documentations
Please select:
